lib/index.js

```diff
@@ -24,10 +24,10 @@ class Oppsy extends Events.EventEmitter {
           psup: ProcessMonitor.uptime,
           psmem: ProcessMonitor.memoryUsage,
           psdelay: ProcessMonitor.delay,
-          requests: this._networkMonitor.requests,
-          concurrents: this._networkMonitor.concurrents,
-          responseTimes: this._networkMonitor.responseTimes,
-          sockets: this._networkMonitor.sockets
+          requests: this._networkMonitor.requests.bind(this._networkMonitor),
```

Comment:
@Marsup @arb Binding the methods like this in constructor also works.

https://gist.github.com/bboysathish/c8d816961aa84bbcc6ff

Still dunno this is best way to do.